lib/TH/vector/AVX.c

```diff
@@ -1,4 +1,3 @@
-#if defined(__AVX__)
```

Comment:
Though it is harmless to remove the macro check for msvc, I think it can be kept because I am not sure the behavior of other platforms. Besides, this macro is the  source I found that caused the compilation broken. #991